meshtastic/tests/test_init.py

```
@@ -218,12 +218,16 @@ def session_passkey(self) -> bytes:
         @session_passkey.setter
         def session_passkey(self, _value: bytes) -> None:
             """Reject writes to simulate an immutable field."""
-            raise RuntimeError("session_passkey is read-only")  # noqa: TRY003 - intentional immutable sentinel
+            raise RuntimeError(
+                "session_passkey is read-only"
+            )  # noqa: TRY003 - intentional immutable sentinel
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)